### ISSUE TITLE
Failed to activate the atom-ide-ui package

### ISSUE THREAD

**Issue by cnurre** [facebook-atom/atom-ide-ui#14]
[Enter steps to reproduce:]

1. Using xatom-syntax 0.3.0 and xatom-ui 0.2.0 
2. Install atom-ide-ui package
3. Get the following error
4. Same happens if I open a file or directory
5. Doesn't seem to happen with other themes

**Atom**: 1.21.0-beta0 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [atom-ide-ui](https://github.com/facebook-atom/atom-ide-ui) package 0.3.0

### Stack Trace

Failed to activate the atom-ide-ui package

```
At variable @font-size is undefined in /Users/chris/.atom/packages/atom-ide-ui/modules/nuclide-commons-ui/styles/CodeSnippet.less:10:15

LessError: variable @font-size is undefined
  at /packages/atom-ide-ui/modules/nuclide-commons-ui/styles/CodeSnippet.less:10:15
```

### Commands

```
```

### Non-Core Packages

```
atom-ide-ui 0.3.0 
close-on-left 0.9.2 
goto-definition 1.3.4 
language-gherkin 1.0.4 
language-lua 0.9.11 
xatom-syntax 0.3.0 
xatom-ui 0.2.0 
```

**Comment by hansonw** [comment]
Thanks for the report. The `@font-size` variable seems to be something that's declared across all the default Atom themes and others like Material UI. I'm inclined to think that this is something `xatom-ui` should declare here:

https://github.com/willyelm/xatom-ui/blob/master/styles/ui-variables.less#L68

For comparison:

https://github.com/atom/one-dark-ui/blob/master/styles/ui-variables.less#L83
https://github.com/atom-material/atom-material-ui/blob/master/styles/ui-variables.less#L78

Although there's no official spec for what `ui-variables` should contain, this seems like a reasonably standard thing to have.

**Comment by cnurre** [comment]
Admittedly, after digging a bit deeper and trying a few more 3rd party themes after submitting this issue it does appear you're right. This is probably an issue with Xatom. Thanks for quick response.

**Comment by almostsultry** [comment]
I get a slight variation on this:

LessError: variable @button-background-color-selected is undefined
  at /Users/captbackfire/.atom/packages/atom-ide-ui/modules/nuclide-commons-ui/styles/Table.less:112:16

**Comment by damieng** [comment]
@almostsultry This is a problem with the theme you are using.

**Comment by jonyeezs** [comment]
Have the same problem as well
Using Seti UI theme
and New Moon syntax theme

**Comment by dotmagic** [comment]
@jonyeezs I'm using seti theme too. Switching to "Atom Dark" Theme resolved the issues for me. I suggest to close this one.

**Comment by forestgagnon** [comment]
@jonyeezs @dotmagic I just opened a PR on seti-ui which resolved the issue for me: https://github.com/jesseweed/seti-ui/pull/459

**Comment by anthonybrown** [comment]
@forestgagnon , how? I'm using seti for the UI theme but I'm still getting the error

**Comment by camlafit** [comment]
Hello

I get also this error. I use adwait pro ui theme (https://github.com/forivall/adwaita-pro-ui/). Error is come with latest atom-ide update

**Comment by parasyte** [comment]
I got this error with a different variable. But my syntax theme already defines the variable:

```
LessError: variable @syntax-background-color is undefined
  at /Users/parasyte/.atom/packages/atom-ide-ui/modules/nuclide-commons-ui/styles/LinePreview.less:6:23
```

The variable is defined here: https://github.com/parasyte/pretty-pastels-syntax/blob/master/styles/syntax-variables.less#L10

**Comment by parasyte** [comment]
Found that this is still related to the UI theme, since mine doesn't import `syntax-variables`, I get this error. The core themes [`atom-dark-ui`](https://github.com/atom/atom-dark-ui/blob/master/styles/ui-variables.less) and [`atom-light-ui`](https://github.com/atom/atom-light-ui/blob/master/styles/ui-variables.less) are affected by this bug as well.

The core themes [`atom-one-dark-ui`](https://github.com/atom/one-dark-ui/blob/master/styles/ui-variables-custom.less#L5) and [`atom-one-light-ui`](https://github.com/atom/one-light-ui/blob/master/styles/ui-variables-custom.less#L5) both import `syntax-variables`, and they both work fine with `atom-ide-ui`.

Adding `@import "syntax-variables";` to the UI theme is a workaround. But note there are core themes missing this.

**Comment by parasyte** [comment]
Oh nice, there is a PR that fixes the issue I described -> https://github.com/facebook-atom/atom-ide-ui/pull/160 Just awaiting release.

**Comment by Arcanemagus** [comment]
This should be fixed here by https://github.com/facebook-atom/atom-ide-ui/pull/160.

**Comment by hansonw** [comment]
Sorry, just published v0.8.1 to fix this!

**Comment by Deuxis** [comment]
Can confirm, I had the issue even though I use Atom Dark (for both window and syntax), updating atom-ide-ui to v0.8.1 fixed it. Nice work!
